src/directedGraph.js

```diff
@@ -0,0 +1,92 @@
+export default class DirectedGraph {
+  constructor(){
+    this.vertices = 0
+    this.edges = []
+    this.edgeCounter = 0
+  }
+  makeGraph(){
+    for( let i = 0; i < this.vertices; ++i ) {
+      this.edges[i] = []
+      this.edges[i].push(i)
+    }
+  }
+
+  addVertex(v) { ++this.vertices }
+  hasVertex(v) {
+    let k
+    for(let i = 0; i < this.edges.length; i++){
+      if(i === v){
+        return true
+      }
+    }
+    return false
+  }
```

Comment:
You create variable 'k' but never used it.  Also, for addVertex, you are only increasing the length or quantity variable of the directed graph instance, and not actually adding it into some sort of object or array?